spec/views/records/_form.html.erb_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   let(:resource) do
-    Image.new
+    ValidatedAssetRepository.new(Image).new
```

Comment:
This seems weird - you changed what a resource is, but the tests didn't need any updates?  I'd have thought you'd have a test for the hint div (if it is a div) or something.